components/SkillsGrid.jsx

```diff
@@ -0,0 +1,17 @@
+export default function SkillsGrid({ skilldata }) {
+  return (
+    <div className="cmp-skills-grid">
+      <h2 className="cmp-skills-grid__heading">Skills</h2>
+      <div className="obj-skills-grid">
```

Comment:
https://www.youtube.com/watch?v=ULdkpU51hTQ

This was shared with me by Kasey a month or so ago. From an accessibility standpoint, we might be able to switch out the parent div with a section kinda like:

```
<section aria-labelledby="Skills-Header" className="cmp-skills-grid"> 
            <h2 id="Skills-Header" className="cmp-skills-grid__heading">Skills</h2> 
```

Reply:
I listened to both with voice over and yeah this is the better way to go.